#request
float64[] CPParams
int8 realtimeTrack
---
#response
bool success
